src/components/BranchMenu.tsx

```diff
@@ -111,6 +118,11 @@ export interface IBranchMenuProps {
    * Git extension data model.
    */
   model: IGitExtension;
+
+  /**
+   * The application language translator.
+   */
+  trans?: TranslationBundle;
```

Comment:
This brings error in the tests because if it is undefined, its method `__` does not exist. I would recommend forcing this parameter to be defined (aka remove the `?`). It is up to the top component to pass the translator or nulltranslator.